docs/DafnyRef/Expressions.md

```diff
@@ -1024,11 +1024,11 @@ function GhostF(z: Stuff): int
 
 The syntax using `:-` is discussed in the following subsection.
 
-## Let or Fail Expression
+## 22.40. Let or Fail Expression
 
 The Let expression described in [Section 0](#sec-let-expression) has a failure variant
```

Comment:
It looks like the other Section 0 links were updated. Wanted to double check this should be left as-is.

Reply:
I have a tool that inserts section numbers automatically. But running it all the time makes for voluminous merges. I run it as a separate pass.